yb-voyager/src/query/sqltransformer/sqlfiletransformer.go

```diff
@@ -51,9 +51,9 @@ type IndexFileTransformer struct {
 	ModifiedIndexesToRange []string
 }
 
-func NewIndexFileTransformer(redundantIndexesToRemove map[string]string, skipPerformanceOptimizations bool, sourceDBType string) *IndexFileTransformer {
+func NewIndexFileTransformer(redundantIndexesToRemove *utils.StructMap[*sqlname.ObjectNameQualifiedWithTableName, string], skipPerformanceOptimizations bool, sourceDBType string) *IndexFileTransformer {
```

Comment:
Q: What was the need/reason to use StructMap here instead of basic map

Reply:
In case redundant change is not applied, I am using `redundantIndexesToRemove` in the report area to add this info to the report so the key of struct `sqlname.ObjectNameQualifiedWithTableName` helps in populating that information easily and the code is readable